src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -462,8 +462,15 @@ module.exports = function<T, P, I, TI, C, CX>(
   }
 
   function bailoutOnLowPriority(current, workInProgress) {
-    if (workInProgress.tag === HostPortal) {
-      pushHostContainer(workInProgress.stateNode.containerInfo);
+    switch (workInProgress.tag) {
+      case ClassComponent:
```

Comment:
If we do this I think we should handle `HostComponent` here too for consistency.
It also [gets popped](https://github.com/bvaughn/react/blob/ed3debfbe2a3d446ddc8755e6013238d4b812ca2/src/renderers/shared/fiber/ReactFiberCompleteWork.js#L201) unconditionally in `complete` phase.
This [defensive check](https://github.com/bvaughn/react/blob/ed3debfbe2a3d446ddc8755e6013238d4b812ca2/src/renderers/shared/fiber/ReactFiberHostContext.js#L129) is likely why it didn't break so far but we can replace it with an assertion.